### PR TITLE
upgrade version of Management.ResourceManager and Management.Storage

### PR DESCRIPTION
**Pull request by erich-wang**
Upgrade version of Management.ResourceManager and Management.Storage to remove local nuget package for management plane.

### CODE REVIEW

**Comment by brjohnstmsft** [comment]
@erich-wang For Azure Search, the data plane tests also depend on fixtures defined in the management plane tests project. Please ensure these tests continue to pass with your change.

**Comment by brjohnstmsft** [comment]
FYI @miwelsh

File: eng/mgmt/AzSdk.test.reference.props [hunk 1]
```diff
@@ -5,7 +5,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.1, 2.0.0)" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.7.5, 2.0.0)" />
-    <PackageReference Include="Microsoft.Azure.ResourceManager" Version="[1.1.0-preview]" />
```
Comment (weshaggard) [inline eng/mgmt/AzSdk.test.reference.props]: Can you delete the checked in packages now?

**Comment by pakrym** [comment]
Is this PR still active? Can we close it if not?

**Pull request closed without merging**